kolibri/plugins/user/assets/src/state/constants.js

```diff
@@ -1,9 +1,9 @@
 
 // a name for every URL pattern
 const PageNames = {
-  LOGIN: 'LOGIN',
+  SIGN_IN: 'SIGN_IN',
   SIGN_UP: 'SIGN_UP',
-  EDIT: 'EDIT',
+  PROFILE: 'PROFILE',
   SCRATCHPAD: 'SCRATCHPAD',
```

Comment:
feel free to get rid of the scratchpad if it's 'in the way', or keep it if it's useful